src/index.css

```diff
@@ -41,3 +41,15 @@ h6 {
 #__next {
   isolation: isolate;
 }
+
+/* html{
+  font-size:1.1vw;
+} */
+@font-face { 
+  font-family: "Pixeloid"; 
+  src: url("../fonts/PixeloidSansBold-PKnYd.ttf"); 
+}
+@font-face { 
+  font-family: "JetBrainsMono"; 
+  src: url("../fonts/JetBrainsMono-VariableFont_wght.ttf"); 
+}
```

Comment:
Already defined in App.css file. No need of it here.

Reply:
The changes are done bhaiya.